apps/dashboard/app/javascript/packs/files/clip_board.js

```diff
@@ -0,0 +1,171 @@
+import ClipboardJS from 'clipboard'
+import Handlebars from 'handlebars';
+import {CONTENTID} from './data_table.js';
+import {EVENTNAME as SWAL_EVENTNAME} from './sweet_alert.js';
+import {EVENTNAME as FILEOPS_EVENTNAME} from './file_ops.js';

```

Comment:
Shouldn't these be plural to indicate there are many?

Reply:
It could, but I don't think it's big enough to change.  that's probably a little too picky. ;)